### ISSUE TITLE
Retina display

### ISSUE THREAD

**Issue by asatormm** [assertthat/selenium-shutterbug#17]
I have a problem on retina display, screen looks like this
![gitgub](https://cloud.githubusercontent.com/assets/23742301/20629299/0d8b11d0-b32b-11e6-8e1d-a1c2834d8712.png)

on non retina is ok 

![github](https://cloud.githubusercontent.com/assets/23742301/20629314/217cf442-b32b-11e6-8a84-9842d6e535eb.png)

```
driver.get("https://github.com");
Shutterbug.shootPage(driver, ScrollStrategy.BOTH_DIRECTIONS).save();

```

**Comment by glib-briia** [comment]
Hi @asatormm,
Could you please update with the following:
- Device you are running on (real/emulator)
- Device screen resolution and pixel ratio
- Selenium version
- Driver version and name
Thanks,
Glib

**Comment by asatormm** [comment]
Device: MacBook Pro (Retina, 13-inch), real
Device screen resolution: 2560x1600, pixel ratio: 2.0
Chrome Version: 54.0.2840.98
Selenium Version: Version 2.53.0
OS: masOS Sierra 10.12.1
Java: 1.8
code:
```
public class screenshot {

    private static WebDriver driver;

    @BeforeTest
    public void setUp() throws Exception {
        System.setProperty("webdriver.chrome.driver", "./libs/chromedriver_mac");
        DesiredCapabilities cap = DesiredCapabilities.chrome();
        driver = new ChromeDriver(cap);
    }

    @Test
    public void infakt_pl() throws IOException{
        driver.get("https://github.com");
        Shutterbug.shootPage(driver, ScrollStrategy.BOTH_DIRECTIONS).save();
    }


    @AfterTest
    public void close() {
        driver.quit();
    }
}
```

**Comment by asatormm** [comment]
@glib-briia @inha-briia 
Could you please fix this issue.
Thanks

**Comment by glib-briia** [comment]
Hi @asatormm,

I'll have a look over the weekend.

**Comment by glib-briia** [comment]
Need to:
-  get window.devicePixelRatio in run time and re-calculate coordinates accordingly for retina only 
- avoid re-calculating while running on emulator, since it will mess things up.

to be done in v. 0.5

**Comment by glib-briia** [comment]
Hi @asatormm ,

Fixed in v0.5 

Usage:
```
Shutterbug.shootPage(driver, ScrollStrategy.BOTH_DIRECTIONS,500,true).save();
```
**true** in the arguments list stand for devicePixelRatio on retina displays.
Thanks,
Glib
